src/ngraph/runtime/cpu/mkldnn_emitter.hpp

```diff
@@ -601,6 +968,7 @@ namespace ngraph
                     {
                         return 4;
                     }
+                    return -1;
```

Comment:
shouldn't this be an assert?

Reply:
do we have something like llvm_unreachable ([link](https://llvm.org/docs/ProgrammersManual.html#programmatic-errors)) to document regions of code that should never be executed, like this one?